modules/veteran_verification/app/controllers/veteran_verification/v0/veteran_status_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module VeteranVerification
+  module V0
+    class VeteranStatusController < ApplicationController
+      before_action { authorize :emis, :access? }
+
+      def index
+        render json: @current_user, serializer: VeteranVerification::VeteranStatusSerializer
+      rescue StandardError
+        raise_error!
+      end
+
+      private
+
+      def raise_error!
+        raise Common::Exceptions::BackendServiceException.new(
+          'EMIS_HIST502',
```

Comment:
Is this some known / documented code from somewhere?

Reply:
Good catch this is not the right thing to return.